mail/template_mailer.py

```diff
@@ -31,7 +32,8 @@ def create_mail(
         recipients: [str],
         text: str,
         cc_recipients: [str]=[],
-        mimetype: str='text'
+        mimetype: str='text',
+        pdfs=[],
```

Comment:
so PDFs are dicts w/ {pdf_bytes, filename}?

I would prefer to make this one tiny bit more generic: add mimetype to the dict (or rather: define a dataclass..). That way, other attachments could also be passed